pinot-common/src/main/java/org/apache/pinot/common/response/broker/BrokerResponseNative.java

```diff
@@ -43,7 +43,7 @@
 public class BrokerResponseNative implements BrokerResponse {
   public static final BrokerResponseNative EMPTY_RESULT = BrokerResponseNative.empty();
   public static final BrokerResponseNative NO_TABLE_RESULT =
-      new BrokerResponseNative(QueryException.BROKER_RESOURCE_MISSING_ERROR);
+      new BrokerResponseNative(QueryException.SERVER_TABLE_MISSING_ERROR);
```

Comment:
This may cause incompatibility with previous versions

Reply:
Thanks for pointing this out.  I will add details to the release notes section.